### DISCUSSION TITLE
cant get past this

### DISCUSSION THREAD

**Discussion by CelestialAriel** [dnschneid/crouton#4926] [Ideas]
```
Please paste the output of the following command here: sudo edit-chroot -all
```

#### Please describe your issue:
I cant get past this IDK what to do and could use some help. trying to install kali linux after doing this on this HP Chromebook thats in developer mode

#### If known, describe the steps to reproduce the issue:
~$ sudo install -Dt /home/ariel/Downloads/bin -m 755 ~/Downloads/cr
outon
install: omitting directory '/home/ariel/Downloads/crouton'
ariel@penguin:~$ sudo crouton
sudo: crouton: command not found
ariel@penguin:~$ sudo apt install crouton
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package crouton
ariel@penguin:~$ sudo crouton
sudo: crouton: command not found
ariel@penguin:~$ sudo install crouton
install: missing destination file operand after 'crouton'
Try 'install --help' for more information.
ariel@penguin:~$ sudo install -cp /home/ariel/Downloads/bin -m 755 ~/Downloads/crouton
install: omitting directory '/home/ariel/Downloads/bin'

**Comment by DennisLfromGA** [comment]
@CelestialAriel,

It appears that you are trying to install `crouton` in a penguin container.
That will not work, crouton is a chroot and relies on ChromeOS processes to run which are not available in a VM & container.

If you already have a working penguin container with a Debian distro you most likely do not *need* a `crouton` chroot also.

*Hope this helps,*
-**DennisLfromGA**

**Comment by CelestialAriel** [comment]
So how would I install Kali Linux on it?
its an hp Chromebook  x360 
Its not the best so I thought If I could make it into a good linux operating laptop it'd be worth it

**Comment by DennisLfromGA** [comment]
@CelestialAriel,

Since you're familiar with Crostini and containers I would stick with that method instead of crouton since it does not require Developer mode. You can install kali Linux distros via the lxc command at: https://us.lxd.images.canonical.com/
See the reddit post linked below for how-to install other Linux distros in Crostini, it uses Ubuntu as an example:

- [How to run Ubuntu with full Chrome OS Integration](https://www.reddit.com/r/Crostini/wiki/howto/run-ubuntu/)

If you instead want to use Developer mode and crouton then you can install the kali-rolling release with crouton's `-r` option. refer to the [README](https://github.com/dnschneid/crouton#hey-now-ubuntu-1604-is-pretty-old-im-young-and-hip) for help with that.


*Hope this helps,*
-**DennisLfromGA**
